.arc

```diff
@@ -1,5 +1,5 @@
 @app
-myproj
+my-enhance-project
```

Comment:
Check the CLI. I may be keying in on the old name to write the .arc file.

Reply:
Yeah, this line in the CLI will need to change when we update the starter-project version in the CLI

https://github.com/beginner-corp/cli/blob/main/src/commands/new/action.js#L65

Reply:
Ah okay. good catch!
My testing has been done via `npm create`. I will revert this portion. And we can discuss at Begin/Enhance meeting how we want to bring this together better